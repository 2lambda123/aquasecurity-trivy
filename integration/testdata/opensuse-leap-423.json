[
  {
    "Target": "testdata/fixtures/opensuse-leap-423.tar (opensuse.leap 42.3)",
    "Vulnerabilities": null
  }
]
