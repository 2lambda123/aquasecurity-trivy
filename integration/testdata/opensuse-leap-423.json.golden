[
  {
    "Target": "testdata/fixtures/images/opensuse-leap-423.tar.gz (opensuse.leap 42.3)",
    "Type": "opensuse.leap"
  }
]